composer.json

```
@@ -286,7 +286,7 @@
         "spryker/util-number": "^1.0.0",
         "spryker/uuid": "^1.0.1",
         "spryker/warehouse-oauth-backend-api": "^1.0.0",
-        "spryker/warehouse-users-backend-api": "^1.0.1",
+        "spryker/warehouse-users-backend-api": "1.1.0",
```

```suggestion
        "spryker/warehouse-users-backend-api": "^1.1.0",
```

please use the instructions from delivery channel in slack